diff -Naur ncurses-6.1-20180908/configure ncurses-6.1-20180908.tpg/configure
--- ncurses-6.1-20180908/configure	2018-09-01 19:37:24.000000000 +0000
+++ ncurses-6.1-20180908.tpg/configure	2018-09-20 16:18:48.920021391 +0000
@@ -328,6 +328,10 @@
   | --runs=* | --run=* | --ru=*)
     runstatedir=$ac_optarg ;;
 
+  --localedir=*)
+    # passed by rpm
+    ;;
+
   -mandir | --mandir | --mandi | --mand | --man | --ma | --m)
     ac_prev=mandir ;;
   -mandir=* | --mandir=* | --mandi=* | --mand=* | --man=* | --ma=* | --m=*)
