--- ncurses-6.0-20150725/configure.omv~	2015-07-27 21:15:15.942908038 +0200
+++ ncurses-6.0-20150725/configure	2015-07-27 21:16:15.760922236 +0200
@@ -5469,7 +5469,7 @@ do
 	done
 	test -n "$cf_add_1lib" && cf_add_libs="$cf_add_libs $cf_add_1lib"
 done
-LIBS="$cf_add_libs"
+LIBS="$cf_add_libs -ldl"
 
 	echo "$as_me:5470: checking whether able to link to dl*() functions" >&5
 echo $ECHO_N "checking whether able to link to dl*() functions... $ECHO_C" >&6
